resources/kcp/charts/kyma-environment-broker/values.yaml

```diff
@@ -81,7 +81,23 @@ osbUpdateProcessingEnabled: "false"
 
 brokerService:
   displayName: "Kyma Environment"
```

Comment:
please offer some solution that we can override any field frm schema by yaml, e.g.:

```
services.ID.description
services.ID.metadata.longDescription
plans.ID.description
plans.ID.metadata.displayName
```